csrc/device_lower/pass/scalar_hoist.cpp

```diff
@@ -453,17 +467,17 @@ std::pair<int64_t, bool> findAllocPointFromDataDependency(
 }
 
 // Inserts allocations of hoisted indices
-class CommonIndexInserter : private kir::ExprMutator {
+class CommonScalarInserter : private kir::ExprMutator {
```

Comment:
Just give this a better name, because we are not only hoisting indices but many other things as well.